src/widgets/refinement-list/defaultTemplates.js

```diff
@@ -1,7 +1,7 @@
 export default {
   header: '',
   item: `<label class="{{cssClasses.label}}">
-  <input type="checkbox" class="{{cssClasses.checkbox}}" value="{{name}}" {{#isRefined}}checked{{/isRefined}} />{{name}}
+  <input type="checkbox" class="{{cssClasses.checkbox}}" value="{{name}}" {{#isRefined}}checked{{/isRefined}} />{{{highlighted}}}
```

Comment:
This can be seen as a possible breaking change, I would make it only for the new template of search values (and so duplicate the template and provide a searchForFacetValues.templates.item). What do you think?